### PR TITLE
Recognize rdf:dirLangString

### PR DESCRIPTION
**Pull request by afs**
See #59.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-semantics/pull/60.html" title="Last updated on Dec 20, 2024, 9:46 AM UTC (3be347c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-semantics/60/a9d9890...3be347c.html" title="Last updated on Dec 20, 2024, 9:46 AM UTC (3be347c)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -815,8 +815,9 @@ <h2>Literals and datatypes</h2>
         for that datatype.
       </p>
       <p>RDF processors are not required to <a>recognize</a> any datatype IRIs other than
-        <a data-cite="RDF12-CONCEPTS#dfn-language-tagged-string"><code>rdf:langString</code></a>
-        and <a data-cite="XMLSCHEMA11-2#string"><code>xsd:string</code></a>,
+        <a data-cite="XMLSCHEMA11-2#string"><code>xsd:string</code></a>,
```
Comment (afs) [inline spec/index.html:818]: Reordered to put the more common datatype first.

**Review by pfps — CHANGES_REQUESTED** [review]
Needs a discussion of whether Concepts needs to be changed.

**Comment by afs** [comment]
> Needs a discussion of whether Concepts needs to be changed.

What change might be necessary?

**Comment by pfps** [comment]
Concepts doesn't say that any datatype is REQUIRED, which seems to me to be needed if Semantics says that RDF reasoning REQUIRES some datatypes.

**Comment by afs** [comment]
This PR is not about the general issue. 

RDF Semantics does require `xsd:string`, `rdf:langString`, therefore it should require `rdf:dirLangString`. This should have been down when the WG completed changes for base direction. It got missed and this PR ensures it does not get lost.

The wider issue is elsewhere:
https://github.com/w3c/rdf-concepts/issues/60#issuecomment-2538919568

**Comment by pfps** [comment]
I created https://github.com/w3c/rdf-concepts/issues/122 explicitly about whether Concepts should discuss requirements for RDF entailment.

**Review by pfps — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

**Comment by pchampin** [comment]
@afs after reverting #52 per [our resolution today](https://www.w3.org/2024/12/19-rdf-star-minutes.html#4761), I tried to rebased this PR, but the formatting change created a conflict :-/ . I will work on it a little later.

**Comment by pchampin** [comment]
> @afs after reverting #52 per [our resolution today](https://www.w3.org/2024/12/19-rdf-star-minutes.html#4761), I tried to rebased this PR, but the formatting change created a conflict :-/ . I will work on it a little later.

that's now fixed. This PR is rebased on the new main (i.e. after #52 has been reverted)

**Comment by gkellogg** [comment]
> @afs after reverting #52 per [our resolution today](https://www.w3.org/2024/12/19-rdf-star-minutes.html#4761), I tried to rebased this PR, but the formatting change created a conflict :-/ . I will work on it a little later.

If there are no intervening commits to preserve, it might be easier to just reset the head and force-push to eliminate those commits.

**Comment by afs** [comment]
Rebased to current main, preview and diff rebuilt.

Editors - this is ready to merge.

**Review by gkellogg — APPROVED** [review]

**Comment by afs** [comment]
Closing.
There are more areas that need updating for `rdf:dirLangString`.

**Pull request closed without merging**